File: Day5/Java/CardValidator.java
import java.util.*;
import java.util.Scanner;
public class CardValidator{
public static boolean number(String numbers){

	Scanner input = new Scanner(System.in);
	int count = 0;

		
	for(int i = 0; i < cardNumber.length(); i++){
		if(cardNumber.charAt(i) != '-'){
			count++;
	if(count == 16){
	return true;
	}
	if(cardNumber.startsWith("4")){
	return true;
	}
	if(cardNumber.startsWith("5")){
	return true;
	}
	if(cardNumber.startsWith("6")){
	return true;
	}
	}
	}
	return false;
}

}

File: Day5/Python/card_validator.py
def card_validator(card_number):

	number = 0
	count = 0

	for numbers in card_number:
		count += 1
		
	
	if count == 16:
		return True
	if card_number.split("-"):
		return True
	if card_number.startswith("4"):
		return True
	if card_number.startswith("5"):
		return True
	if card_number.startswith("6"):
		return True
	else:
		return False
